azerothcore/azerothcore-wotlk
# chore(DB/transports): Replace transport `name` fields with consistent text.

## Changes Proposed:
- Replaces the `name` field for each entry in the `transports` table with a consistent standard.

The area the transport patrols `(if two points, sort by areatable ID)` (Transport type, faction ("Transport name" `(if named)`))
Example: `Orgrimmar, Durotar and Thunder Bluff, Mulgore (Zeppelin, Horde ("The Zephyr"))`

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

Wowpedia for The Zephyr transport name.

## Tests Performed:

- Ran the query, no errors.

## How to Test the Changes:

1. Run the query.
2. Make sure the transports still transport I guess.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
